docs/technical_reference/named_releases.rst

```diff
@@ -377,7 +377,7 @@ Aspen
 
 * **Code cut date:** 2014-09-05
 * **Status:** unsupported
-* `Release Notes <https://edx.readthedocs.io/projects/open-edx-release-notes/en/latest/aspen.html>`__
+* Release notes: Not available
```

Comment:
@nedbat: FYI: This was getting a 404.

Reply:
Hmm, yeah, looks like that never existed.